src/control/lib/control/system_test.go

```diff
@@ -1143,11 +1143,12 @@ func TestControl_EventForwarder_OnEvent(t *testing.T) {
 	rasEventRankDown := events.NewRankDownEvent("foo", 0, 0, common.NormalExit)
 
 	for name, tc := range map[string]struct {
-		aps            []string
-		event          *events.RASEvent
-		nilClient      bool
-		expInvokeCount int
-		expErr         error
+		aps             []string
+		event           *events.RASEvent
+		nilClient       bool
+		expInvokeCount  int
+		expNextSequence uint64
```

Comment:
Should we always assume expNextSequence == (expInvokeCount + 1)? If so maybe it'd be better to derive it inside the test logic